std_msgs/Header header

int32 episode

string state
